erpnext/patches/v13_0/add_default_statement_of_account_email_template.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+import os
+
+import frappe
+from frappe import _
+
+
+def execute():
+	frappe.reload_doc("email", "doctype", "email_template")
+	frappe.reload_doc("accounts", "doctype", "accounts_settings")
+
+	if not frappe.db.exists("Email Template", _("Statement of Account")):
+		base_path = frappe.get_app_path("erpnext", "templates", "emails")
+		response = frappe.read_file(os.path.join(base_path, "statement_of_account_email_notification.html"))
+
+		frappe.get_doc({
+			"doctype": "Email Template",
+			"name": _("Statement of Account"),
+			"response": response,
+			"subject": _("Statement of Account"),
```

Comment:
```suggestion
			"subject": "Statement of Account",
```

Reply:
![Estado-de-cuenta](https://user-images.githubusercontent.com/6947417/97525301-77458680-19cc-11eb-8f9f-b15cd679a5fb.png)

Reply:
this needed to translate

Reply:
- You do not need to explicitly put translation in db
- While rendering the text, frappe checks for translation available in the system, if translation is found, it'll display the translation.
- So if we insert `Statement of Account` in db, while displaying to the user, system will display it in translated language only.
- Insert translated text is not the recommended way.
- I know that similar code was already committed by frappe tech itself, but that practise has been discontinued now